tests/test_consolidation.py

```diff
@@ -40,6 +41,28 @@ def forward(self, data):
         return out
 
 
+@pytest.mark.parametrize("ngpu", ["single", "multi"])
```

Comment:
I feel that single and multi are testing the same thing because this model is not sharded?